src/engraving/libmscore/lyrics.cpp

```diff
@@ -319,8 +319,8 @@ void Lyrics::layout()
 //                   LOGD("create leading, trailing <%s> -- <%s><%s>", qPrintable(text), qPrintable(leading), qPrintable(trailing));
             const TextBlock& tb = textBlock(0);
 
-            const qreal leadingWidth = tb.xpos(leading.size(), this) - tb.boundingRect().x();
-            const int trailingPos = text.size() - trailing.size();
+            const qreal leadingWidth = tb.xpos(static_cast<int>(leading.size()), this) - tb.boundingRect().x();
+            const int trailingPos = static_cast<int>(text.size() - trailing.size());
```

Comment:
Looks like it's better to update TextBlock::xpos to use size_t, doesn't it?

Reply:
Probably, but that'd be for another PR?

Reply:
Or another commit ;-)